source/adapters/cuda/image.cpp

```diff
@@ -492,9 +502,11 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesImageFreeExp(
 UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesUnsampledImageCreateExp(
     ur_context_handle_t hContext, ur_device_handle_t hDevice,
     ur_exp_image_mem_handle_t hImageMem, const ur_image_format_t *pImageFormat,
-    const ur_image_desc_t *pImageDesc, ur_mem_handle_t *phMem,
+    const ur_image_desc_t *, ur_mem_handle_t *,
```

Comment:
What's the purpose of this change? I see elsewhere in this function that you're making a change similar to what's being done in https://github.com/oneapi-src/unified-runtime/pull/1512. 

However, the change here isn't quite the same. Maybe a bad rebase? Either way, I don't think this PR needs to or should depend on the aforementioned one.

The same applies to `urBindlessImagesSampledImageCreateExp`.

Reply:
I didn't want to explicitly depend on https://github.com/oneapi-src/unified-runtime/pull/1512 since that would introduce a lot of noise in my PR, and make it harder to review. But this PR does indeed depend on https://github.com/oneapi-src/unified-runtime/pull/1512

Reply:
Knowing that keeping `phMem` around would require non-trivial changes, I'm more than happy to include the removal here. Resolving.